### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by spavlusieva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,14 +55,23 @@ If you would like to consider contributing to the project, please see our [Contr
...
 
 As of the beginning of this repo, contributions have been made by
 [Victor Cacciari Miraldo](https://github.com/VictorCMiraldo), [Harold Carr](https://github.com/haroldcarr), [Mark Moir](https://github.com/mark-moir), and [Lisandra Silva](https://github.com/lisandrasilva), all while employed at Oracle Labs.
 
 We are grateful to the following people for contributions since then:
 * [Chris Jenkins](https://github.com/cwjnkins)
+
+## Security
+
+Please consult the [security guide](./SECURITY.md) for our responsible security vulnerability disclosure process
```
Comment (mark-moir) [inline README.md:70]: Missing period.

File: README.md [hunk 2]
```diff
@@ -55,14 +55,23 @@ If you would like to consider contributing to the project, please see our [Contr
...
+
+## License
+
+Copyright (c) 2020, 2023 Oracle and/or its affiliates.
```
Comment (mark-moir) [inline README.md:74]: Should this be 2020-2023, or maybe 2020, 2021, 2022, 2023, or is it OK as is?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,50 +1,55 @@
...
-by your fix.
-4. Ensure that your changes have not broken any existing proofs (`./Scripts/run-everything.sh yes` should complete without errors).
-5. Update the [list of contributors](README.md#contributors) to include your name if it does not already.
-6. Submit a pull request. *Do not leave the pull request blank*. Explain exactly
-what your changes are meant to do and provide simple steps on how to validate
-your changes. Ensure that you reference the issue you created as well.
-7. We will review the pull request before it is merged.
+## Code of conduct
 
-## Code of Conduct
```
Comment (mark-moir) [inline CONTRIBUTING.md:48]: This capitalisation is inconsistent with the others you've changed, but maybe it's intentional?

**Review by mark-moir — APPROVED** [review]
Thanks @spavlusieva!  Just a couple of minor comments.  APPROVED after you've addressed them (or not) to your satisfaction.

File: README.md (hunk 1 above)
Comment (spavlusieva) [reply README.md]: thanks. added

File: README.md (hunk 2 above)
Comment (spavlusieva) [reply README.md:74]: according to https://github.com/oracle/template-repo/blob/main/README.md:
The correct copyright notice format for both documentation and software is "Copyright (c) [year,] year Oracle and/or its affiliates."
You must include the year the content was first released (on any platform) and the most recent year in which it was revised

**Review by mark-moir — APPROVED** [review]
LGTM, thanks @spavlusieva.  Merging.

**Pull request merged**